AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -110,6 +110,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+scaleway.com#%#//scriptlet('set-cookie', 'is_gdpr_banner_closed', 'true')
```

Comment:
It's worked ok on my end since the first pr was merged.
It seems we can use a simpler rule, please check it `scaleway.com##footer ~ div.didiff-hide`.

Reply:
Checked. https://github.com/AdguardTeam/AdguardFilters/pull/98802/commits/6a5745e24e07fb6a82cb59b78fdcdfea556eba21